### DISCUSSION TITLE
Opschalingsanalyse vraag 6: Hoe kan het sensorenregister worden geïmplementeerd en wat zijn de randvoorwaarden en risico’s?

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#257] [Pilot Gemeente]
Opschalingsanalyse vraag 6: **Hoe kan het sensorenregister worden geïmplementeerd en wat zijn de randvoorwaarden en risico’s?**

**Comment by WOostdam** [comment]
Niet elke gemeente zijn eigen omgeving laten installeren/huren, maar met een aantal landelijke of provinciale hubs werken. Zodat gemeenten die niet de kennis hebben om zelf zo'n platform te installeren of de hele mikmak te beheren wel mee kunnen doen. Uiteraard tegen een redelijke vergoeding. Deze grootschaligheid zorgt ook voor lagere kosten per deelnemer.
Gemeenten die dat willen en kunnen (betealen) kunnen natuurlijk wel hun eigen omgeving neerzetten.

Wil het echt een groot succes worden, dan moet "iedereen" (alle overheden) meedoen en moet het een verplicht karakter hebben. Ook allemaal gebruik maken van dezelfde tooling, zodat er eenvormige en eenduidige data opgebouwd worden (geen varianten of smaakjes).

**Comment by marcvanandel** [comment]
Dit betreft diverse gemeentelijke (en algemene) architectuurvraagstukken. Daarvoor is een aparte discussion: #245 Daar heb ik al diverse perspectieven en onderwerpen benoemd ...
